### ISSUE TITLE
phenogram/ideogram viewer

### ISSUE THREAD

**Issue by nlwashington** [monarch-initiative/monarch-legacy#261]
I want this integrated in our site...i want to see phenotypes on chromosomes.

http://visualization.ritchielab.psu.edu/phenograms/document

It seems like the Ritchie lab have done a lot if hard work, and provide this for anyone to use.

**Comment by cmungall** [comment]
Is that the same as this - http://www.ebi.ac.uk/fgpt/gwas/

Can we just leapfrog straight to jbrowse?

**Comment by nlwashington** [comment]
Based on that, but an actual embeddable toolkit in d3. Chatted with someone
from the group. They are in Pennsylvania...

**Comment by mellybelly** [comment]
wonder how configurable, would be nice if you could filter ontologically... lets discuss soon. I think we should try to have genome views in july release

**Comment by harryhoch** [comment]
Nicole, they said it was written in D3? Interesting. hard to tell via the site.

The win would be in coordination and interaction. Imagine the ideogram integrated with our model viewer….

On Apr 7, 2014, at 10:23 PM, Melissa Haendel notifications@github.com wrote:

> wonder how configurable, would be nice if you could filter ontologically... lets discuss soon. I think we should try to have genome views in july release
> 
> —
> Reply to this email directly or view it on GitHub.

---

Harry Hochheiser
University of Pittsburgh
Department of Biomedical Informatics 
harryh@pitt.edu         412 648 9300

**Comment by nlwashington** [comment]
Yep.  I briefly talked with Sarah (who presented the work-http://ritchielab.psu.edu/ritchielab/people/current-members/sarah-pendergrass/), and they actually have the software open source and would be thrilled if we used/embedded it into our website.  It is highly configurable (at least that is what was shown), and would be awesome.  I am unclear exactly how "responsive" it is (we didn't see a live demo, just screenshots), but it is under active development.

Harry, the url i saw actually had "d3" in it, so i suspect it was written with that toolkit (but i don't know for 100%).

Anyway, Sarah is excited to talk to us further.  She had to run off to catch her flight, so I only got a 3-min-bathroom-run-in to chat with her.

Nicole

**Comment by harryhoch** [comment]
So the Ritchie lab tool looks like it generates PNG images. The views are nice, but they are static.-I wasn’t able to find any information about D3 anywhere there. 

The EBI GWAS browser renders everything in SVG, but not D3. they have a nice mouse-over to get the specific annotation for each variant, and it’s all ontologically-linked, via the EFO. Might be interesting to see what we could do to build on this foundation - particularly given that we could probably ask Helen for some guidance and/or code.

An ideal phenogram viewer would, I think, include better facilities for navigating to a  single chromosome, gene, or even exon - as Chris mentioned, more or less devolving at some point into a sequence browser. Making that transition seamless might not be trivial, but it could be interesting.

On Apr 8, 2014, at 2:31 AM, Nicole Washington notifications@github.com wrote:

> Yep. I briefly talked with Sarah (who presented the work-http://ritchielab.psu.edu/ritchielab/people/current-members/sarah-pendergrass/), and they actually have the software open source and would be thrilled if we used/embedded it into our website. It is highly configurable (at least that is what was shown), and would be awesome. I am unclear exactly how "responsive" it is (we didn't see a live demo, just screenshots), but it is under active development. 
> 
> Harry, the url i saw actually had "d3" in it, so i suspect it was written with that toolkit (but i don't know for 100%). 
> 
> Anyway, Sarah is excited to talk to us further. She had to run off to catch her flight, so I only got a 3-min-bathroom-run-in to chat with her. 
> 
> Nicole 
> 
> On Apr 7, 2014, at 7:48 PM, Harry Hochheiser wrote: 
> 
> > Nicole, they said it was written in D3? Interesting. hard to tell via the site. 
> > 
> > The win would be in coordination and interaction. Imagine the ideogram integrated with our model viewer…. 
> > 
> > On Apr 7, 2014, at 10:23 PM, Melissa Haendel notifications@github.com wrote: 
> > 
> > > wonder how configurable, would be nice if you could filter ontologically... lets discuss soon. I think we should try to have genome views in july release 
> > > 
> > > — 
> > > Reply to this email directly or view it on GitHub. 
> > 
> > ---
> > 
> > Harry Hochheiser 
> > University of Pittsburgh 
> > Department of Biomedical Informatics 
> > harryh@pitt.edu 412 648 9300 
> > — 
> > Reply to this email directly or view it on GitHub. 
> > 
> > —
> > Reply to this email directly or view it on GitHub.

---

Harry Hochheiser
University of Pittsburgh
Department of Biomedical Informatics 
harryh@pitt.edu         412 648 9300

**Comment by nlwashington** [comment]
http://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/human/issues/?filters=fix-version:grch38p3,grch38p2,grch38p1+scaffold-type:novel,fix

for us, instead of ticket types, we could use this for phenotypes, diseases, variants, etc. 

there's search.  faceting.  barcharts.  and a table to see the relevant data.

THIS IS WHAT I WANT.

**Comment by harryhoch** [comment]
yup. I think we could take this even further. See, for instance…

http://www.cs.ubc.ca/labs/imager/tr/2013/VariantView/
On May 4, 2015, at 6:56 PM, Nicole Washington notifications@github.com wrote:

http://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/human/issues/?filters=fix-version:grch38p3,grch38p2,grch38p1+scaffold-type:novel,fix

for us, instead of ticket types, we could use this for phenotypes, diseases, variants, etc.

there's search. faceting. barcharts. and a table to see the relevant data.

THIS IS WHAT I WANT.

—
Reply to this email directly or view it on GitHubhttps://github.com/monarch-initiative/monarch-app/issues/261#issuecomment-98879858.

---

Harry Hochheiser
University of Pittsburgh
Department of Biomedical Informatics
harryh@pitt.edumailto:harryh@pitt.edu  412 648 9300

**Comment by nlwashington** [comment]
@nathandunn and @selewis you will be interested in that link http://www.cs.ubc.ca/labs/imager/tr/2013/VariantView/ for apollo

**Comment by nathandunn** [comment]
Thanks.  Some really nice visualizations.   I like how they’d optimized it for a dense number of variants, it would be worth emulating.  

Also important to note is that they had in-line list views, similar to what we are adding into Apollo.  

Nathan
